src/components/pages/StatedGraphForm.tsx

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
+import Chart from '../Chart';
+import { observer } from 'mobx-react-lite';
+import BalanceService from '../BalanceService';
 
-import ChartFunctional from '../ChartFunctional';
-let GraphForm = () => {
+const GraphForm = (balance) => {
   return (
     <div>
       <h1>GraphPage</h1>
-      <ChartFunctional
-        values={[
-          [new Date('December 17, 2020 03:24:00'), 20.2],
-          [new Date('December 17, 2020 06:24:00'), 20.2],
-          [new Date('December 19, 2020 12:24:00'), 42.2],
-          [new Date('December 20, 2020 12:24:00'), 53.2]
-        ]}
+      <Chart
+        values={
+          balance.balance
```

Comment:
it is strange that this function gets balances and needs to access balance of those balances.
Something is fishy here.
balanceService.getBalances should already return array of balances.